objcgen/embedder.cs

```diff
@@ -290,7 +292,7 @@ public int Compile ()
 					break;
 				case Platform.iOS:
 					build_infos = new BuildInfo[] {
-					new BuildInfo { Sdk = "iPhoneOS", Architectures = new string [] { "armv7", "armv7s", "arm64" }, SdkName = "iphoneos", MinVersion = "8.0", XamariniOSSDK = "MonoTouch.iphoneos.sdk" },
+					new BuildInfo { Sdk = "iPhoneOS", Architectures = new string [] { "armv7", "armv7s", "arm64" }, SdkName = "iphoneos", MinVersion = "8.0", XamariniOSSDK = "MonoTouch.iphoneos.sdk", CompilerFlags = "-fembed-bitcode", LinkerFlags = "-fembed-bitcode" },
```

Comment:
Why is this needed? It looks unrelated.

Reply:
Default "new" xcode iOS project are defaulting bitcode to on, so it's likely an attempt to shut that up.

Reply:
looks like my other pull request has got merged into this pull